apps/evans/components/pages/document/DownloadAudiobook.tsx

```diff
@@ -29,134 +31,183 @@ const DownloadLinks: React.FC<Props> = ({
+            </Link>
+            <Link
+              href="/app"
+              className="text-xs normal-case bg-flprimary text-white rounded-full -mt-1 ml-4 px-6 py-1"
+            >
+              {t`Recommended`}
+            </Link>
+          </dt>
+          <dd className="text-flgray-500 text-xs mb-4 pb-1">
+            <Dual.Frag>
+              <>
+                (Listen <span className="hidden sm:inline">on your phone</span> with our
+                iOS or Android app)
+              </>
+              <>
+                <span className="sm:hidden">
+                  Escucha con nuestra aplicación iOS o Android
+                </span>
+                <span className="hidden sm:inline">
+                  Escucha en tu teléfono con la aplicación iOS o Android
+                </span>
+              </>
+            </Dual.Frag>
+          </dd>
+          <dt className="uppercase text-md mb-1">
+            <a href={podcastLoggedDownloadUrl[quality]} className="hover:underline">
+              <span className="hidden sm:inline">{t`Download as`} </span>Podcast
+            </a>
+          </dt>
+          <dd className="text-flgray-500 text-xs mb-4 pb-1">
+            (Apple Podcasts, Stitcher,{` `}
+            <span className="hidden sm:inline">Overcast, </span>
+            etc.)
+          </dd>
+          <dt className="uppercase text-md mb-1">
+            <a
+              href={mp3ZipLoggedDownloadUrl[quality]}
+              className="hover:underline"
+              onClick={
+                isSingleAudio
+                  ? async (event) => {
+                      event.preventDefault();
+                      try {
+                        if (!mp3LoggedDownloadUrls[0]) {
+                          throw new Error(`no mp3 url provided`);
+                        }
+                        setDownloading(true);
+                        const downloadMp3Res = await fetch(
+                          mp3LoggedDownloadUrls[0][quality],
+                        );
+                        if (!downloadMp3Res.ok) {
+                          throw new Error(`failed to download mp3`);
+                        }
+                        const blob = await downloadMp3Res.blob();
+                        const url = window.URL.createObjectURL(blob);
+                        const link = document.createElement(`a`);
+                        link.href = url;
+                        link.setAttribute(`download`, title);
+                        document.body.appendChild(link);
+                        link.click();
+                        link.remove();
+                        window.URL.revokeObjectURL(url);
+                        setDownloading(false);
+                      } catch (error) {
+                        setDownloading(false);
+                        window.location.href = mp3ZipLoggedDownloadUrl[quality];
+                      }
+                    }
+                  : () => {
```

Comment:
since we're not preventing default unless it's a single audio, this second function is not necessary. i'm pretty sure you can just pass `undefined` to any `onClick` prop which is basically like not setting it. let's do that instead.